src/Export/UrlBuilderService.php

```diff
@@ -123,7 +123,7 @@ protected function getSalesChannelDomain(): ?string
      */
     protected function getProductSeoPath(ProductEntity $product): ?string
     {
-        $allSeoUrls = $product->getSeoUrls();
+        $allSeoUrls = $this->removeInvalidUrls($product->getSeoUrls()->getElements());
         if (!$allSeoUrls) {
```

Comment:
This `null` check is still needed and has no purpose in the current state

Reply:
So you need to:
- Still check if `$product->getSeoUrls()` returns something
- Additionally check if `removeInvalidUrls` does return an empty collection

Reply:
@TobiasGraml11 
Guess only one condition is needed?
Then it is enough?: `if (!count($allSeoUrls))`
Not needed to check productSeoUrls too:`if (!count($product->getSeoUrls()) || !count($allSeoUrls))`

Reply:
You will need something like this:
```php
if (!$product->getSeoUrls()) {
    return null;
}

$allSeoUrls = $this->removeInvalidUrls($product->getSeoUrls())

if (!$allSeoUrls->count()) {
    return null;
}
```

Reply:
We also need to check count() on $product->getSeoUrls(), correct?

```
if (!$product->getSeoUrls()->count()) {
    return null;
}
```

Reply:
No, because the code will not throw an exception in that case